letsencrypt/Dockerfile

```diff
@@ -18,8 +19,20 @@ RUN apk add --no-cache --update \
         libffi-dev \
         musl-dev \
         openssl-dev \
+    && pip3 install setuptools==57.5.0 \
+    && echo "cffi==1.14.6" >> /tmp/requirements.txt \
+    && echo "cryptography==3.4.8" >> /tmp/requirements.txt \
+    && echo "setuptools==57.5.0" >> /tmp/requirements.txt \
+    && apkArch="$(apk --print-arch)"; \
+       case "$apkArch" in \
+         armhf) export ARCH='armhf' ;; \
+         x86) export ARCH='i386' ;; \
+         armv7) export ARCH='armv7' ;; \
+         x86_64) export ARCH='amd64' ;; \
+         aarch64) export ARCH='aarch64' ;; \
+        esac \
     && pip3 install --no-cache-dir --find-links \
-        "https://wheels.home-assistant.io/alpine-$(cut -d '.' -f 1-2 < /etc/alpine-release)/${BUILD_ARCH}/" \
+        "https://wheels.home-assistant.io/alpine-$(cut -d '.' -f 1-2 < /etc/alpine-release)/${ARCH}/" \
```

Comment:
Why you change that?

Reply:
Watching the build I saw zero prebuilt wheels being pulled from the home assistant wheel repository. When looking into it further it seemed the arch type was always resolving to "empty".

Reply:
Could be because the ARCH arg is missing above

Reply:
Do you mean the BUILD_ARCH? I was not able to get it properly working, maybe it was something to do with my pipeline? Regardless I have tested the current implementation and it works fine for all the architectures.

Reply:
Right, but we use BUILD_ARCH and the issue is, it's just not added on the file there. You can't use an arg if that is not specified before. That is the issue, I don't say you did the issue

Reply:
My misunderstanding, sorry!
If I am getting you correctly I should be able to add:
ARG BUILD_ARCH 
then I can revert the URL to use ${BUILD_ARCH} and get rid of this entire section
`apkArch="$(apk --print-arch)"; \
       case "$apkArch" in \
         armhf) export ARCH='armhf' ;; \
         x86) export ARCH='i386' ;; \
         armv7) export ARCH='armv7' ;; \
         x86_64) export ARCH='amd64' ;; \
         aarch64) export ARCH='aarch64' ;; \
        esac \`

I am guessing that gets passed as part of the CI build process?

Reply:
right

Reply:
I'll try to push that up later tonight.

Reply:
pushed up the changes we discussed